crates/matrix-sdk-crypto/src/backups/mod.rs

```diff
@@ -463,57 +463,4 @@ mod test {
 
         backup_flow(machine).await
     }
-
-    #[async_test]
-    #[cfg(feature = "sled_cryptostore")]
-    async fn default_store_backups() -> Result<(), OlmError> {
-        use tempfile::tempdir;
-
-        let tmpdir = tempdir().expect("Can't create a temporary dir");
-        let machine =
-            OlmMachine::with_default_store(alice_id(), alice_device_id(), tmpdir.as_ref(), None)
-                .await?;
-
-        backup_flow(machine).await
-    }
-
-    #[async_test]
-    #[cfg(feature = "sled_cryptostore")]
-    async fn recovery_key_storing() -> Result<(), OlmError> {
```

Comment:
#558 moves this test into the integration tests instead of just removing it. The other `sled_cryptostore` dead code is also removed in #558.

Reply:
Okay, I'll remove this commit then.